test/tests/materials/derivative_sum_material/random_ic.i

```diff
@@ -3,28 +3,15 @@
   dim = 2
   nx = 10
   ny = 10
-  nz = 0
-  xmin = 0
   xmax = 250
-  ymin = 0
   ymax = 250
-  zmin = 0
-  zmax = 0
   elem_type = QUAD4
 []
 
 [Variables]
   [./c]
-    order = FIRST
-    family = LAGRANGE
     [./InitialCondition]
-      type = SmoothCircleIC
-      x1 = 125.0
-      y1 = 125.0
-      radius = 80.0
-      invalue = 1.0
-      outvalue = 0.1
-      int_width = 80.0
+      type = RandomIC
```

Comment:
FYI @dschwen  I pulled this test out of phase_field by changing this IC. I think this still tests the DerivativeSumMaterial in the same way, and it keeps the test with the src.